requirements.txt

```diff
@@ -41,3 +41,6 @@ python-dateutil
 
 # Requesting resources
 pybiomart
+
+# Required by scripts
+tabulate
```

Comment:
I mean this! 😄

Reply:
Aaah OK!

Reply:
I'm always afraid of inflating the requirements - dependencies can be a pain.

Reply:
I think it's safe not include the script dependencies in the requirements. I don't think anybody except us will ever use this script. And I think if we'll use it again it will be in a long time